services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -170,7 +170,8 @@ class DynamicSidecarEgressSettings(BaseCustomSettings):
         description="envoy image to use",
     )
     DYNAMIC_SIDECAR_ENVOY_LOG_LEVEL: EnvoyLogLevel = Field(
-        default=EnvoyLogLevel.ERROR, description="log level for envoy proxy service"
+        default=EnvoyLogLevel.ERROR,  # type: ignore
```

Comment:
this mypy I also see and find a bit strange. Did you search why this is happenning?

Reply:
Yes I did search, but I have no solution for it. If you look at the enum, it's using an `auto()` to compute the value. Somehow I can't manage to make it work with any typing trick.